docker-compose.yml

```diff
@@ -16,11 +16,10 @@ services:
       - uwpath_db_network
       - uwpath_backend_network
     environment:
+      - DB_HOST=db
+      - SECRET_KEY=7ldnlu_gz*&vx&=4)q3#74ih@nzx*owgeu^=*katjqpbb-s1sk
```

Comment:
Is it ok to expose this secret key like this?

Reply:
this should be okay, we could throw env variable in there.